src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
```

Comment:
This line is a bit complex and could be simplified. Instead of using `slice` and `split` to remove the dollar sign and commas from the salary, you could use a regular expression with the `replace` method. This would make the code easier to read and more robust.